src/witan/models/run_models.clj

```diff
@@ -164,14 +164,14 @@
                            :proportion-male-newborns
                            (double (/ (:number-male-newborns user-parameters)
                                       (:number-all-newborns user-parameters))))
-                    (catch Exception e (println (format "Caught Error: %s"
-                                                        (.getMessage e)))))]
-    (println (format "%d option validation errors\n%s" (count errors)
-                     (clojure.string/join "\n" errors)))
-    (println (format "\nInput used: %s\nOutput used: %s\n"
-                     (:input-config (:options (parse-opts args cli-options)))
-                     (:output-projections (:options (parse-opts args cli-options)))))
-    (println (format "Preparing projection for %s... " (get-district gss-code)))
+                    (catch Exception e (info (format "Caught Error: %s"
+                                                     (.getMessage e)))))]
+    (info (format "\n%d option validation errors\n%s" (count errors)
```

Comment:
Yeah that's right, info's what we want for this sort of thing specifically for the demo tool